.pre-commit-config.yaml

```diff
@@ -20,3 +20,10 @@ repos:
     - id: bandit
       language_version: python3
       exclude: ^tests/
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.740
+    hooks:
+    - id: mypy
+      language_version: python3
+      # To prevent "Duplicate module named 'setup'" error
```

Comment:
I was arguing about this with Anthony a while back. Still now sure about the best way to prevent this: https://github.com/pre-commit/mirrors-mypy/issues/5.

Reply:
Would you suggest this solution? https://github.com/pre-commit/mirrors-mypy/issues/5#issuecomment-515841288

Reply:
Maybe. But then, I think, I've had problems with that too. It's up to you atm.

Reply:
I think there's nothing wrong with this exception since packages are raw test fixtures. I'd prefer to leave it as is and move on. It's unlikely that type checking in packages could be any useful.